src/Componet/Home.jsx

```diff
@@ -1,10 +1,51 @@
 // import Eu from "../imagrnd/Eu.jpg"
+import { useState } from "react"
 import home from "../Componet/home.css"
 
 export default function Home() {
+    const [cor,setcor]= useState(1);
+
+    const preto={color:"#000"};
+    const cinza={color:"#acacac"};
+    const marron={color:"#965aas"}
+
+    const retCor=(c)=>{
+        if(cor==1){
+          return  preto
+        } else if(cor==2){
+            return cinza
+        }else{
+            return marron
+        }
+    }
+
+    const mudarCor=()=>{
+        setcor(cor+1)
+        if(cor>2){
+            setcor(1)
+        }
+    }
+
+    setInterval(mudarCor,3000)
```

Comment:
todo código assíncrono (que é executado depois de um certo tempo) deve estar em um useEffect

```suggestion
    useEffect(() => {
        setInterval(mudarCor,3000)
    }, [mudarCor])
```

a função _mudarCor_, como foi usada como dependência de um _useEffect_, deve ser declarada com um _useCallback_. veja https://react.dev/reference/react/useCallback